config/detekt.yml

```diff
@@ -117,6 +117,9 @@ mozilla-detekt-rules:
     excludes: "**/*Test.kt, **/*Spec.kt, **/test/**, **/androidTest/**"
   MozillaCorrectUnitTestRunner:
     active: true
+  MozillaRunBlockingCheck:
+    active: true
+    excludes: "**/*Test.kt"
```

Comment:
nit: we should copy the same exclusion blocks as all the others. e.g. we don't want to enforce this in `androidTest` either.